IdentityCore/tests/MSIDLRUCacheTest.m

```diff
@@ -379,7 +379,7 @@ - (void)testMSIDLRUCache_whenCallingAPIsUseThrottlingCacheWithinGCDBlocks_thrott
     [expectation4 fulfill];
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->